packages/java-edition/src/mcfunction/node/argument.ts

```diff
@@ -138,6 +139,13 @@ export namespace EntitySelectorAtVariable {
 			value as EntitySelectorAtVariable,
 		)
 	}
+
+	export function filterAvailable(ctx: core.ContextBase) {
+		const release = ctx.project['loadedVersion'] as ReleaseVersion | undefined
+		return EntitySelectorAtVariables.filter(variable =>
+			!(variable === '@n' && release && ReleaseVersion.cmp(release, '1.21'))
```

Comment:
am i tripping?
```suggestion
			!(variable === '@n' && release && ReleaseVersion.cmp(release, '1.21') < 0)
```

Reply:
Not sure how this happened, thanks for catching this!